x/protocolpool/keeper/keeper_test.go

```diff
@@ -160,3 +160,62 @@ func (s *KeeperTestSuite) TestIterateAndUpdateFundsDistribution() {
 	})
 	s.Require().NoError(err)
 }
+
+func (suite *KeeperTestSuite) TestGetCommunityPool() {
+	suite.SetupTest()
+
+	expectedBalance := sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, math.NewInt(1000000)))
+	suite.authKeeper.EXPECT().GetModuleAccount(suite.ctx, types.ModuleName).Return(poolAcc).Times(1)
+	suite.bankKeeper.EXPECT().GetAllBalances(suite.ctx, poolAcc.GetAddress()).Return(expectedBalance).Times(1)
+
+	balance, err := suite.poolKeeper.GetCommunityPool(suite.ctx)
+	suite.Require().NoError(err)
+	suite.Require().Equal(expectedBalance, balance)
+
+	// Test error case when module account doesn't exist
+	suite.authKeeper.EXPECT().GetModuleAccount(suite.ctx, types.ModuleName).Return(nil).Times(1)
+	_, err = suite.poolKeeper.GetCommunityPool(suite.ctx)
+	suite.Require().Error(err)
+	suite.Require().Contains(err.Error(), "module account protocolpool does not exist")
+}
+
+func (suite *KeeperTestSuite) TestSetToDistribute() {
```

Comment:
I think @facundomedica  is going to change the behavior of this (or maybe only in begin blocker). ACK for now, but probably this test will have to change.